src/PaymentHandler/PayoneIDealPaymentHandler.php

```diff
@@ -36,7 +38,7 @@ public static function isCapturable(array $transactionData, array $payoneTransAc
             return false;
         }
 
-        $txAction = isset($transactionData['txaction']) ? strtolower((string) $transactionData['txaction']) : null;
```

Comment:
did you run ecs or did your IDE the formatting? please remove this change, if it is not done by ecs.